backend/services/implementations/postingService.ts

```diff
@@ -138,23 +147,30 @@ class PostingService implements IPostingService {
           },
         },
       );
-      return postings.map((posting: PostingWithRelations) => {
-        return {
-          id: String(posting.id),
-          branch: convertToBranchResponseDTO(posting.branch),
-          shifts: convertToShiftResponseDTO(posting.shifts),
-          skills: convertToSkillResponseDTO(posting.skills),
-          employees: convertToEmployeeResponseDTO(posting.employees),
-          title: posting.title,
-          type: posting.type,
-          status: posting.status,
-          description: posting.description,
-          startDate: posting.startDate,
-          endDate: posting.endDate,
-          autoClosingDate: posting.autoClosingDate,
-          numVolunteers: posting.numVolunteers,
-        };
-      });
+
+      return await Promise.all(
+        postings.map(async (posting: PostingWithRelations) => {
+          const employeesArr: EmployeeUserResponseDTO[] = await convertToEmployeeUserResponseDTO(
```

Comment:
Please rename all other usages as well

```suggestion
          const employeeUsers EmployeeUserResponseDTO[] = await convertToEmployeeUserResponseDTO(
```